client/components/formatted-header/docs/example.jsx

```diff
@@ -0,0 +1,18 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import FormattedHeader from '..';
+
+export default function FormattedHeaderExample() {
```

Comment:
I went ahead and changed this to a class with `displayName`. I didn't expect there to be much advantage to that, but [apparently it's important](https://github.com/Automattic/wp-calypso/commit/a11d5092dabb7d519071128f1e0cd8ba43f76698). The [README] should probably say that explicitly _adds todo_.

[README]: https://github.com/Automattic/wp-calypso/blob/95d1c2d937810a5bc3a8a63a54ec941304c2c609/client/devdocs/design/README.md#component-name-convention